spec/multi_database_spec.cr

```diff
@@ -0,0 +1,24 @@
+require "./spec_helper"
+
+private class ModelWithBadDatabase < BaseModel
+  table do
+  end
+
+  # Use a different database for this fake model
+  def self.database
+    DatabaseWithIncorrectSettings
+  end
+end
+
+describe "Configuring and connecting to different databases" do
+  it "tries to connect to the configured database" do
+    # It will fail to connect which is what we expect since we configured
+    # a database with an incorrect URL
+    #
+    # If does not raise an error then that means it is using the good connection,
+    # which is not what we configured
+    expect_raises Avram::ConnectionError do
```

Comment:
Should this raise an exception error message letting us know which one it wasn't able to connect to? Our app could have up to 4 PG connections technically (a bit absurd, I know), so if for some reason one of them didn't connection, it would be nice to see `Unable to connect with DatabaseWithIncorrectSettings` or something.

Reply:
Yes great point!